packages/cozy-harvest-lib/src/components/KonnectorConfiguration/KonnectorConfiguration.jsx

```diff
@@ -82,8 +82,13 @@ class KonnectorConfiguration extends React.Component {
     const hasLoginError = hasError && konnectorJobError.isLoginError()
     const hasTermsVersionMismatchError =
       hasError && konnectorJobError.isTermsVersionMismatchError()
+
+    const hasConsistentTermsVersionMismatchError =
+      hasTermsVersionMismatchError &&
+      konnectorsModel.hasNewVersionAvailable(konnector)
```

Comment:
Why do we need to do that ? Is it related to banking connectors ?

Reply:
It has been used for banking konnector, but it is not specific to them. The case when this happens is explained in [the comment above](https://github.com/cozy/cozy-libs/pull/731#discussion_r318661364)